wallet/src/nightfall-browser/event-handlers/block-proposed.js

```diff
@@ -154,37 +160,38 @@ async function blockProposedEventHandler(data, zkpPrivateKeys, nullifierKeys) {
     }),
   );
 
+  let height = 1;
```

Comment:
We reuse this multiplee times across the codebase, maybe it is a good candidate to pt it to common-files at this point?